packages/components/src/components/hds/form/radio-card/index.js

```diff
@@ -4,29 +4,14 @@
  */
 
 import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
-import { action } from '@ember/object';
 import { assert } from '@ember/debug';
-import { schedule } from '@ember/runloop';
-import { setAriaDescribedBy } from '../../../../utils/hds-set-aria-described-by.js';
 
 export const DEFAULT_CONTROL_POSITION = 'bottom';
 export const DEFAULT_ALIGNMENT = 'left';
 export const CONTROL_POSITIONS = ['bottom', 'left'];
 export const ALIGNMENTS = ['left', 'center'];
 
 export default class HdsFormRadioCardIndexComponent extends Component {
-  @tracked ariaDescribedBy = this.args.extraAriaDescribedBy;
-  @tracked descriptors = [];
-
-  @action
-  setAriaDescribedBy() {
-    // we schedule this afterRender to capture all descriptors registered onInsert
-    schedule('afterRender', () => {
-      setAriaDescribedBy(this);
-    });
-  }
-
```

Comment:
None of this seems to actually be used by the component. It seems like maybe a copy/paste from another component but it doesn't look like this has ever been used by this component: https://github.com/hashicorp/design-system/commit/5edb999bae52a5f1bf1dad68a983e57d7993e3b5